README.md

```diff
@@ -42,6 +47,11 @@ On Windows, one may need to specify generator:
 
     cmake -G "Visual Studio 15 2017" ..
 
+If the SQLite3 dependency is installed in a custom location, specify the
+paths to the include directory and the library::
+
+    cmake -DSQLITE3_INCLUDE_DIR=/opt/SQLite/include -DSQLITE3_LIBRARY=/opt/SQLite/lib/libsqlite3.so ..
```

Comment:
Or use the `CMAKE_PREFIX_PATH` cache variable (which is a list BTW):
~~~
cmake -DCMAKE_PREFIX_PATH=/opt/SQLite ..
~~~

This will also help find the executable (`find_program`).

Reply:
I've never had problems finding `sqlite3` (it's on the `PATH`). Just the other components.

This suggestion works too, so I've added it to docs/source/install.rst